schema.sql

```diff
@@ -1,10 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100),
-    name: string,
-    date_of_birth: date,
-    escape_attempts: integer,
-    neutered: boolean,
-    weight_kg: decimal,
+    id INT,
+    name TEXT,
+    date_of_birth DATE,
+    escape_attempts INT,
+    neutered BOOLEAN,
+    weight_kg DECIMAL
 );
```

Comment:
[OPTIONAL] You can improve your table by making the id column not null and autoincremented. Autoincrementing the ID field will make sure that an ID is always generated every time a new record is created even if it is not inserted manually.
```
id int NOT NULL IDENTITY(1,1)
```
as well as make it a primary key to ensure row-level accessibility.
```
PRIMARY KEY (id)
```